Documentation/dev/libvirt-howto.md

```diff
@@ -123,39 +129,45 @@ include the `--permanent` to the commands that add-source and add-port.
     4. Set the `name` (e.g. test1)
     5. Look at the `podCIDR` and `serviceCIDR` fields in the `networking` section. Make sure they don't conflict with anything important.
     6. Set the `pullSecret` to your JSON pull secret.
```

Comment:
needs doc on where to get this file.  Also @derekwaynecarr showed me it's easier to point to a path than to try to inline the file:
pullSecretPath: "/home/bparees/git/gocode/src/github.com/openshift/installer/files/config.json"

Reply:
(the reason i say we need doc on where to get this file is i was told *not* to get it from coreos.com (and i ran into problems trying to do so).

Reply:
> `pullSecretPath: "/home/bparees/git/gocode/src/github.com/openshift/installer/files/config.json"`

This [is deprecated][1].  Once we get the new installer and #320, you'll be able to use `OPENSHIFT_INSTALL_PULL_SECRET="$(cat path/to/your/secret)"`.

[1]: https://github.com/openshift/installer/blob/6eb2377149820ce13dfc7f8bb4414ade028ada2c/pkg/types/config/cluster.go#L85